CHANGELOG.md

```diff
@@ -5,10 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
```

Comment:
No need to introduce more whitespace here (unless I'm missing something?) Stupid Github won't let me suggest the change right away :upside_down_face: